site/pages/experimental-fr.hbs

```diff
@@ -242,3 +255,42 @@ if ( s.getTime() &lt; msT && msT &lt; e.getTime() ) {
 		</div>
 	</div>
 </div>
+
+<h3 id="beta-alert">Beta alert designs</h3>
```

Comment:
@Borges-Porteous Can we remove the beta mention? Since it's already in a provisional process

Reply:
Addressed in #1650. I removed all mentions of "beta" with regards to the alerts.